package.json

```diff
@@ -51,7 +51,7 @@
     "ecstatic": "^2.1.0",
     "es2040": "^1.2.3",
     "mutant": "github:ahdinosaur/mutant#request-idle-callback",
-    "patchbay": "^7.3.0",
+    "patchbay": "^7.3.1",
```

Comment:
we want to remove the `^`, which is saying "the latest version which matches 7.x.y"